packages/jazz-tools/src/tests/coPlainText.test.ts

```diff
@@ -63,6 +113,11 @@ describe("CoPlainText", () => {
         expect(text.length).toBe(11);
       });
 
+      test("as string", () => {
+        const text = CoPlainText.create("hello world", { owner: me });
+        expect(`${text}`).toBe("hello world");
+      });
```

Comment:
nit: can we add also a test for `as number`? Just for better code coverage.

Reply:
I've tweaked the output to use `Number(RawCoPlainText)` to allow numbers to be returned appropriately